tests/conftest.py

```diff
@@ -39,7 +39,7 @@ def input_var_check_ttnn(request):
 
 @pytest.fixture(scope="session")
 def device():
-    device = ttnn.open_device(device_id=0)
+    device = ttnn.open_device(device_id=0, l1_small_size=1024)
```

Comment:
How did you choose this l1_small_size?

https://github.com/search?q=repo%3Atenstorrent%2Ftt-metal+l1_small_size&type=code&p=2

Reply:
I forgot to mention this. I simply searched for the smallest size that can pass all model tests.

Is there a better way to determine the size? I found https://github.com/tenstorrent/tt-metal/issues/13746 mention currently it needs to be decided manually

Reply:
I see. We can go with this way now.

I will flag this to the team to determine a proper solution. 
This potentially makes allocator more inefficient.

Reply:
I added a TODO to link to https://github.com/tenstorrent/tt-metal/issues/13746